doc/jsk_pcl_ros/nodes/organized_pass_through.rst

```diff
@@ -23,7 +23,7 @@ Publishing Topic
 
 -  ``~output`` (``sensor_msgs/PointCloud2``)
 
-  Filtered point cloud.
+  Filtered point cloud where ``NaN`` points are removed.
```

Comment:
@wkentaro O, I now understand what you mean.
This nodelet uses `pcl::ExtractIndices` with `setKeepOrganized` is true and I removed indices whose point values are `NaN` to extract.
It works well and I didn't mind what is replaced with NaN value in filtered pointcloud.
However, i found that by default removed points are replaced to `NaN` value!
http://docs.pointclouds.org/trunk/classpcl_1_1_filter_indices.html#a9456a457b18c28b8dd6b07e970c16eba
Very weird...